### DISCUSSION TITLE
Sync Email Labels and use as whitelist/blocklist

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#4221] [Ideas]
Would be nice to sync gmail labels and be able to use this as a whitelist/blocklist instead.

Received from a user:

<img width="830" alt="Screenshot 2024-02-28 at 13 08 09" src="https://github.com/twentyhq/twenty/assets/6399865/0cdbf49d-c137-4a6d-a445-75b483b8c154">

**Comment by BOHEUS** [comment]
@neo773 how big effort would be to allow synchronizing emails by labels? https://developers.google.com/workspace/gmail/api/guides/labels

**Reply by neo773** [reply]
We already do? Labels = Folders in Gmail driver
